ci/cron/src/Main.hs

```diff
@@ -366,11 +370,13 @@ check_releases gcp_credentials bash_lib max_releases = do
                   let gcp_path = "gs://daml-data/releases/" <> v <> "/github/" <> f
                   exists <- does_backup_exist gcred bash_lib gcp_path
                   if exists then do
-                      putStrLn $ gcp_path <> " already exists."
+                      let local = (temp_dir </> f <> ".gcp")
+                      gcs_cp gcred bash_lib gcp_path local
+                      check_files_match gcp_path local >>= \case
```

Comment:
IIUC this calls `diff` on a remote path like `gs://daml-data/releases/...`.

Reply:
Yes, I completely screwed up. Hopefully fixed up version coming on.